app/grandchallenge/reader_studies/views.py

```diff
@@ -1090,6 +1087,23 @@ def get_serializer_class(self):
             return DisplaySetPostSerializer
         return DisplaySetSerializer
 
+    def create_civ(self, data):
+        interface = data.pop("interface", None)
+        value = data.pop("value", None)
+        image = data.pop("image", None)
+        with transaction.atomic():
+            if interface.is_image_kind:
+                # New images can also be added via the rawimageupload endpoint
+                if image:
+                    civ = ComponentInterfaceValue(interface=interface)
+                    civ.image = image
+                    civ.full_clean()
+                    civ.save()
+                    return civ
+            elif interface.is_json_kind:
+                civ = interface.create_instance(value=value)
+                return civ
```

Comment:
Anything that falls through this if statement is silently dropped, do we really want that to happen?